modules/seqexec/server/src/main/scala/seqexec/server/keywords/package.scala

```diff
@@ -119,10 +137,75 @@ package keywords {
       KeywordBag(ks.toList.map(internalKeywordConvert))
 
   }
+
+  // A simple typeclass to encapsulate default values
+  trait DefaultValue[A] {
```

Comment:
This is kind of a dangerous typeclass because it's lawless and easily abused. Can we give it a more specific name and restrict its usage?

```scala
sealed abstract class DefaultHeaderValue[A] private {
  ...
}
object DefaultHeaderValue {
  // define every instance here
}
```

Reply:
Makes sense